reference/features/filtering_and_faceted_search.md

```diff
@@ -0,0 +1,318 @@
+# Filtering and faceted search
+
+You can use MeiliSearch's filters to refine search results and create faceted search interfaces.
+
+## Configuring filters
+
+Filters use [document fields](/learn/core_concepts/documents.md#fields) to establish filtering criteria.
+
+To use a document field as a filter, you must first add its attribute to the [`filterableAttributes` index setting](/reference/api/filterable_attributes.md).
+
+**This step is mandatory and cannot be done at search time.** Filters need to be properly processed and prepared by MeiliSearch before they can be used, an operation that takes as much time as indexing all your documents.
```

Comment:
```suggestion
**This step is mandatory and cannot be done at search time.** Filters need to be properly processed and prepared by MeiliSearch before they can be used.

Updating `filterableAttributes` requires recreating the entire index. This may take a significant amount of time depending on your dataset size.
```
```